polldaddy-xml.php

```diff
@@ -1,5 +1,6 @@
 <?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
 
+#[AllowDynamicProperties]
```

Comment:
Yeah, from what I saw the XML class is extended by other classes, which add the appropriate attributes.

Still, it seems better to add that annotation just in case. :D